services/application.js

```diff
@@ -303,27 +303,28 @@ class Application {
         }
     }
 
+    async remindApplicationSubmission() {
+        const inactiveDuration = this.emailParams.inactiveDays - this.emailParams.remindDay;
```

Comment:
This logic is incorrect. If use default values, inactiveDays is 45 and remnidDay is 30 so inactiveDuration is 15 and we send a reminder after 15 days. We want to send the reminder after 30 days so we can remove this "inactiveDuration" calculation and just use the remindDay value instead

Reply:
updated it to 30 days.